### DISCUSSION TITLE
Add OSM2World as „Streer-View“ layer

### DISCUSSION THREAD

**Discussion by DerKarlos** [maplibre/maplibre#169] [Cross-project feature proposals]
There is a Web-Frontend (TypeScript) for the 3D renderer OSM2World (not deployed yet) that could be integrated into Maplibre as a street level viewer (as competitors have)

![image](https://user-images.githubusercontent.com/1619669/208441407-717dc9d3-02c5-4e26-8e12-33ec7389b01f.jpeg)

**Comment by wipfli** [comment]
Interesting suggestion. How would the traditional 2d renderer and this 3d renderer blend together?
